x-pack/plugins/observability_solution/observability_onboarding/public/application/quickstart_flows/firehose/index.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import {
+  EuiPanel,
+  EuiSkeletonRectangle,
+  EuiSkeletonText,
+  EuiSpacer,
+  EuiSteps,
+  EuiStepStatus,
+} from '@elastic/eui';
+import useEvent from 'react-use/lib/useEvent';
+import { FETCH_STATUS, useFetcher } from '../../../hooks/use_fetcher';
+import { EmptyPrompt } from '../shared/empty_prompt';
+import { CreateStackCommandSnippet } from './create_stack_command_snippet';
+import { VisualizeData } from './visualize_data';
+
+export function FirehosePanel() {
+  const [windowLostFocus, setWindowLostFocus] = useState(false);
+  const { data, status, error, refetch } = useFetcher((callApi) => {
+    return callApi('POST /internal/observability_onboarding/firehose/flow');
+  }, []);
```

Comment:
I'm getting the access denied error twice (once in the empty prompt and then again in the toast). I think this is the case because you're not passing in { showToastOnError: false } option into the useFetcher hook.